guides/plugins/plugins/administration/using-vuex-state.md

```diff
@@ -0,0 +1,113 @@
+# Using Vuex Stores
```

Comment:
Don't we follow a specific guideline here ?:

```markdown
# Using Vuex Stores

## Overview
[...]

## Prerequisites
```

Reply:
jes we do the `## Overview` probably got lost somewhere